src/libs/ajax/leonardo/LeoRuntimesV1DataClient.ts

```diff
@@ -0,0 +1,40 @@
+import { AbortOption, FetchFn } from 'src/libs/ajax/data-client-common';
+import { RawGetRuntimeItem } from 'src/libs/ajax/leonardo/models/runtime-models';
+
+export interface LeoRuntimesV1DataClient {
```

Comment:
Naming nits:
- This is specific to _Google_ runtimes.
- Thoughts on abbreviated vs full service names (Leo vs Leonardo)?